scripts/check_file_license_apache.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+FLAG=$1
+FILE=$2
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+TIMESCALE_LOCATION=`dirname $SCRIPTPATH`
+
+LICENSE_STRING=
+FIRST_COMMENT=
+
+if [[ "$FLAG" == '-c' ]]; then
+    LICENSE_STRING='/*  * Copyright (c) 2018  Timescale, Inc. All Rights Reserved.  *  * This file is licensed under the Apache License,  * see LICENSE-APACHE at the top level directory.  */ '
```

Comment:
These license strings seem very brittle. Can we export them in a fixed file somewhere as constants? Then if we ever change the wording, it's easier to hunt down.